var/spack/repos/builtin/packages/magma/package.py

```diff
@@ -186,21 +186,40 @@ def post_install(self):
     def cache_test_sources(self):
         """Copy the example source files after the package is installed to an
         install test subdirectory for use during `spack test run`."""
-        self.cache_extra_test_sources([self.test_src_dir])
+        cache_extra_test_sources(self, [self.test_src_dir])
 
-    def test(self):
+    def test_c(self):
+        """Run benchmark tests"""
```

Comment:
Are these really benchmarks?  

```suggestion
        """Run C examples"""
```